lib/_http_outgoing.js

```diff
@@ -407,9 +424,10 @@ Object.defineProperty(OutgoingMessage.prototype, 'headersSent', {
 
 OutgoingMessage.prototype.write = function(chunk, encoding, callback) {
   var self = this;
+  var err;
 
   if (this.finished) {
-    var err = new Error('write after end');
+    err = new Error('write after end');
```

Comment:
I guess that's a fair point.  What's the motivation for emitting the exception as an error instead of throwing it?  Both are going to be disruptive.
 
As a small aside, I notice that OutgoingMessage#write() still returns true, indicating it's okay to call it again.  That seems like a buglet to me.